{"name": "Verify API handles invalid endpoint", "status": "passed", "steps": [{"name": "Given the API client is initialized for TC_01 Third Scenario", "status": "passed", "start": 1739655120458, "stop": 1739655120459}, {"name": "When I send a GET request to an invalid endpoint", "status": "passed", "start": 1739655120459, "stop": 1739655121080}, {"name": "Then the response status should be 404", "status": "passed", "start": 1739655121081, "stop": 1739655121081}], "start": 1739655120457, "stop": 1739655121081, "uuid": "58d81106-8eb9-4f8d-8164-7a9c72461288", "historyId": "ee47dd7b8c3888cdf3c79429f5c48e96", "fullName": "GET User List, User Login and Invalid Endpoint Test: Verify API handles invalid endpoint", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "GET User List, User Login and Invalid Endpoint Test"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}